src/bosh-director/lib/bosh/blobstore_client/s3cli_blobstore_client.rb

```diff
@@ -126,7 +126,7 @@ def sign_url(object_id, verb, duration)
     end
 
     def required_credential_properties_list
-      %w[access_key_id secret_access_key]
+      []
```

Comment:
This needs to be conditional on whether `credentials_source` is `static` or not. We need to validate credentials exist/DNE if the source is `static` still.

Reply:
thanks @h4xnoodle 
I added the logic to validate `static` back in, and added a couple tests.